### PR TITLE
adding my new project

### PR DESCRIPTION
**Pull request by ghost**
Hi!!

I think our project would be great for Up for Grabs. 

Let me know if you have any thoughts or feedback on the template.

### CODE REVIEW

File: _data/projects/MindsDB.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: MindsDB
+desc: An Explainable AutoML framework for developers built on top of Pytorch. I
```
Comment (shiftkey) [inline _data/projects/MindsDB.yml:3]: Typo?

```suggestion
desc: An Explainable AutoML framework for developers built on top of Pytorch.
```

File: _data/projects/MindsDB.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/mindsdb/mindsdb
+tags:
+- python
+- automl
+- xai
+- xai-library
+upforgrabs:
+  name: bug
+  link: https://github.com/mindsdb/mindsdb/issues?q=is%3Aissue+is%3Aopen+label%3Abug
```
Comment (shiftkey) [inline _data/projects/MindsDB.yml:12]: Linking directly to the label makes this link a bit more readable:

```suggestion
  link: https://github.com/mindsdb/mindsdb/labels/bug
```

**Review by shiftkey — COMMENTED** [review]
Thanks for the contribution @amiexdata! 

Just a couple of suggestions to improve the content, everything else is :gem:

File: _data/projects/MindsDB.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/MindsDB.yml:3]: @amiexdata  I think MindsDB is more than putting it in a few words. But still, can we try and make it better in the description? Like `An Explainable autoML framework to streamline the use of neural networks built on top of Pytorch`

File: _data/projects/MindsDB.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+name: MindsDB
+desc: An Explainable AutoML framework for developers built on top of Pytorch. I
+site: https://github.com/mindsdb/mindsdb
+tags:
```
Comment (ritwik12) [inline _data/projects/MindsDB.yml:5]: Can you please add more tags here like `machine learning, pytorch, ludwig, tensorflow, artificial intelligence`

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
Nice to meet you again @amiexdata :)
Small world!!
Lightwood would also be good to add here.

**Comment by shiftkey** [comment]
Closing this out because the user deleted their account
